pkgs/ffigen/lib/src/code_generator/writer.dart

```diff
@@ -390,13 +390,26 @@ class Writer {
             strings.ffiNative: usesFfiNative,
           },
           strings.symbols: {
-            for (final b in bindings) b.usr: {strings.name: b.name},
+            for (final b in bindings) b.usr: _makeSymbolMapValue(b),
           },
         },
       },
     };
   }
 
+  Map<String, String> _makeSymbolMapValue(Binding b) {
+    final dartName = b is Typealias ? getTypedefDartAliasName(b) : null;
```

Comment:
The `is` check out here is redundant. `getTypedefDartAliasName` already returns null if it's not a `Typealias`.

Reply:
The compiler doesn't allow passing a binding as a type (Typealias is a Binding that implements a Type).
Adding a check is therefore neccesarry. (I could also have done `b is Type` instead)